lib/salus/scanners/report_pom_xml.rb

```diff
@@ -0,0 +1,38 @@
+require 'salus/scanners/base'
+
+# Report Java library usage
+
+module Salus::Scanners
+  class ReportPomXml < Base
+    def run
+      shell_return = run_shell("bin/parse_pom_xml #{@repository.path_to_repo}/pom.xml", chdir: nil)
+
+      if !shell_return.success?
+        report_error(shell_return.stderr)
+        return
+      end
+
+      dependencies = JSON.parse(shell_return.stdout)
```

Comment:
It would be safer to check for could not parse json error, in case weird things somehow got appended to stdout.